### ISSUE TITLE
grep_string does not jump to correct line

### ISSUE THREAD

**Issue by krehwell** [nvim-telescope/telescope.nvim#2444]
### Description

I'm using 
```lua
vim.keymap.set("n", "<C-f>", ":Telescope grep_string search=<CR>")
```

when the popup opened, and press `<CR>` to the file I searched by string for, it opens the file correctly but it does not jump my cursor to the correct line as in the popup shown instead it jumps me to first line of the file

### Neovim version

```markdown
NVIM v0.8.3
Build type: Release
LuaJIT 2.1.0-beta3
Compiled by builduser
```

### Operating system and version

6.2.8-arch1-1

### Telescope version / branch / rev

telescope master

### checkhealth telescope

```markdown
telescope: require("telescope.health").check()
========================================================================
## Checking for required plugins
  - OK: plenary installed.
  - OK: nvim-treesitter installed.

## Checking external dependencies
  - OK: rg: found ripgrep 13.0.0
  - OK: fd: found fd 8.7.0

## ===== Installed extensions =====

## Telescope Extension: `fzf`
  - OK: lib working as expected
  - OK: file_sorter correctly configured
  - OK: generic_sorter correctly configured
```

### Steps to reproduce

1. mapped and source
```lua
vim.keymap.set("n", "<C-f>", ":Telescope grep_string search=<CR>")
```

2. press `<C-f>`. The popup will open
3. search for something, ex: `foo`
4. press Enter/`<CR`>
5. notice that it opens the file correctly but does not jump the cursor to the line as shown in the popup during searching

### Expected behavior

once I found what I wanted to search for, pressing enter should open the file and jump my cursor to the correct line as shown in the popup

### Actual behavior

it opens the file and my cursor is at top most of the file

### Minimal config

```Lua
telescope.setup({
	defaults = {
		mappings = {
			i = {
				["<c-["] = actions.close,
				["<esc>"] = actions.close,
			},
		},
	},
	pickers = {
		buffers = {
			initial_mode = "insert",
			ignore_current_buffer = true,
			sort_lastused = true,
			theme = "dropdown",
			previewer = false,
			mappings = {
				i = {
					["<c-d>"] = actions.delete_buffer,
				},
				n = {
					["<c-d>"] = actions.delete_buffer,
				},
			},
			layout_config = {
				width = 115,
				height = 22,
			},
		},
		find_files = {
			theme = "ivy",
			layout_strategies = "vertical",
			layout_config = {
				height = 11,
			},
			hidden = true,
		},
		git_files = {
			theme = "ivy",
			layout_strategies = "vertical",
			layout_config = {
				height = 11,
			},
			hidden = true,
		},
		grep_string = {
			prompt_prefix = "🔍 ",
			no_ignore = true,
			file_ignore_patterns = { "%.png", "%.jpg", "%.jpeg", "%.gif", "%.webp", "%.svg", "%.tsbuildinfo" },
			layout_config = {
				width = 160,
			},
		},
	},
	extensions = {
		fzf = {
			fuzzy = true, -- false will only do exact matching
			override_generic_sorter = true, -- override the generic sorter
			override_file_sorter = true, -- override the file sorter
			override_live_grep_sorter = true,
			case_mode = "smart_case", -- or "ignore_case" or "respect_case", default "smart_case"
		},
	},
})

require("telescope").load_extension("fzf")

vim.keymap.set("n", "<C-f>", ":Telescope grep_string search=<CR>")
```

**Comment by jamestrew** [comment]
I think the issue is your mapping: 
```lua
vim.keymap.set("n", "<C-f>", ":Telescope grep_string search=<CR>")
```
I'm not exactly sure what your intentions are with this mapping but generally, `grep_string` is used to grep for the word under your cursor and fuzzy find over the results - the `search` option is by default set as the word under your cursor (in normal mode). The way you have it mapped, you're not going to be grepping for `foo` in for example.

Either way, I don't think your use case is the intended way to use `grep_string`. I would recommend `live_grep` instead if you want to simply search for a word.

**Comment by krehwell** [comment]
> I'm not exactly sure what your intentions are with this mapping but generally

I map it so, so that I can do live search just like fzf does. It previously working great though, my cursor could jump to the position where the popup was showing the line I'm search for. 

my previous command is like this
```lua
vim.keymap.set(
	"n",
	"<C-f>",
	":lua require'telescope.builtin'.grep_string{ shorten_path = true, word_match = \"-w\", only_sort_text = true, search = '' }<CR>"
)
```

I simplified the command as in this issue desc to make it simpler

**Comment by jamestrew** [comment]
Still feels like a misuse of the picker. Not sure if you would get the support you want in a timely manner. Check out `live_grep` if you haven't done so already.

```
builtin.live_grep({opts})                      *telescope.builtin.live_grep()*
    Search for a string and get results live as you type, respects .gitignore

builtin.grep_string({opts})                  *telescope.builtin.grep_string()*
    Searches for the string under your cursor or the visual selection in your
    current working directory
```

**Comment by krehwell** [comment]
live grep does not find the string in fuzzy manner. 

> Not sure if you would get the support you want in a timely manner

not sure about if it's a bug or never really supported. but it was previously working find, I've been using grep string to find string fuzzyly for over 1/2 years and works greatly

using grep_string to setting up fuzzy live match is also a normal thing in telescope I think, it's what many people suggest anyway here
- https://github.com/nvim-telescope/telescope.nvim/issues/564#issuecomment-786850829
- https://www.reddit.com/r/neovim/comments/rndg7t/how_to_make_telescope_live_grep_support_fzf_fuzzy/

**Comment by jamestrew** [comment]
> live grep does not find the string in fuzzy manner.

Ohh ok gotcha. It's making sense to me now, my bad.

Doing a bit of digging, I think I was the one who actually broke this in https://github.com/nvim-telescope/telescope.nvim/pull/2416 :sweat_smile: 
I'll look into a fix but in the meantime you can switch over to the `0.1.1` tag/branch or this is the SHA of the commit just before that PR got merged: `10ebb30f0de54fe`.

**Comment by krehwell** [comment]
thank you very much, using the branch suggested (`0.1.1`) fix my issue
